paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
Here is my usage, that prevents chrome suggesting.

<paper-typeahead-input
    input-value="{{_selectorValue}}"
    id="query_input"
    local-candidates="[[transactions]]"
    placeholder="Transaction ID"
    autocomplete="Off"
    is-allow-suggest-on-empty-input="true"
    suggest-on-focus="true"
    max-suggestions="10"
>
</paper-typeahead-input>


On 18 July 2016 at 17:16, Tianxiang Zhang <notifications@github.com> wrote:

> In paper-typeahead-input.html
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30#discussion_r71154420>
> :
>
> > @@ -313,7 +340,7 @@
> >          }
> >        });
> >        var input = Polymer.dom(this.root).querySelector('paper-input');
> > -      input.$.input.autocomplete = 'address-level4';
> > +      input.$.input.autocomplete = this.autocomplete;
>
> do u mean you are using autocomplete="off"? because based on your current
> code, it's setting autocomplete="address-level4" still.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30/files/dfdc34b0472906d1f9c7005d4442b0162ab8c4f8#r71154420>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG0SnMfV0Oamco4x7R5UNbHdlgCM5fYQks5qW4qzgaJpZM4JKWUL>
> .
>



-- 
C уважением, Пличко Павел